### PR TITLE
Zonal Heat Pump for Whole-Home Panelized Retrofit

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

File: measures/ResStockArguments/measure.rb [hunk 1]
```diff
@@ -728,7 +740,26 @@ def run(model, runner, user_arguments)
...
     if args['wall_continuous_exterior_r'].is_initialized
-      args['wall_assembly_r'] += args['wall_continuous_exterior_r'].get
+      wall_continuous_exterior_r = args['wall_continuous_exterior_r'].get
+
+      # Optionally derate the continuous exterior wall insulation
+      if args['wall_area'].is_initialized
+        wall_area = args['wall_area'].get
+
+        zonal_heat_pump_wall_area = 6.0 # ft^2
+        zonal_heat_pump_wall_area *= args['geometry_unit_num_bedrooms']
```
Comment (joseph-robertson) [inline measures/ResStockArguments/measure.rb:750]: `num_beds` + 1

Comment (joseph-robertson) [inline measures/ResStockArguments/measure.rb:750]: `num_beds` + 1

File: measures/ResStockArguments/measure.rb [hunk 2]
```diff
@@ -728,7 +740,26 @@ def run(model, runner, user_arguments)
...
+        else
+          zonal_heat_pump_r = wall_continuous_exterior_r
+        end
+        actual_wall_area = args['wall_area'].get - zonal_heat_pump_wall_area
+
+        wall_continuous_exterior_r = (actual_wall_area / wall_area * wall_continuous_exterior_r) + (zonal_heat_pump_wall_area / wall_area * zonal_heat_pump_r)
+        args['wall_continuous_exterior_r'] = wall_continuous_exterior_r
+      end
+
+      args['wall_assembly_r'] += wall_continuous_exterior_r
```
Comment (joseph-robertson) [inline measures/ResStockArguments/measure.rb:762]: Try area-weighting with total assembly r-value.

Comment (joseph-robertson) [inline measures/ResStockArguments/measure.rb:762]: Try area-weighting with total assembly r-value.

**Comment by afontani** [comment]
Closing PR. This was a project.

**Pull request closed without merging**